FEXCore/Source/Interface/Core/Core.cpp

```diff
@@ -724,6 +716,13 @@ namespace FEXCore::Context {
       Thread->ExecutionThread->detach();
     }
 
+    if (Thread->DestroyedByParent) {
+#ifndef _WIN32
+      Alloc::OSAllocator::UninstallTLSData(Thread);
+#endif
+      SignalDelegation->UninstallTLSState(Thread);
```

Comment:
Why does this need to be done now when it seemingly wasn't there before?

Reply:
Oops, this leaked in from the other changes. This isn't yet required for this changed. Removed for now.